tests/foreman/api/test_repository.py

```diff
@@ -1944,6 +1946,91 @@ def test_positive_sync_publish_promote_cv(self):
         self.assertGreaterEqual(len(result.stdout), 1)
 
 
+class SRPMRepositoryIgnoreContentTestCase(APITestCase):
+    """Test whether SRPM content can be ignored during sync.
+
+    In particular sync of duplicate SRPMs would fail when using the flag
+    ``ignorable_content``.
+    """
+
+    @classmethod
+    def setUpClass(cls):
+        """Create a product and an org which can be re-used in tests."""
+        super().setUpClass()
+        cls.org = entities.Organization().create()
+        cls.product = entities.Product(organization=cls.org).create()
+
+    @tier2
+    def test_positive_ignore_sprm_duplicate(self):
+        """Test whether SRPM duplicated content can be ignored.
+
+        :id: de03aaa1-1f95-4c28-9f53-362ceb113167
+
+        :expectedresults: SRPM content is ignored during sync.
+
+        :CaseLevel: Integration
+
+        :CaseComponent: Pulp
+
+        :BZ: 1673215
```

Comment:
Since these tests are in their own class, you can move these metadata field to class docstring. Then you won't have to repeat them for every test.
Or you can leave them for each test case, it doesn't matter for final results.